src/screens/accountSettings/EditConfirmations.js

```diff
@@ -0,0 +1,75 @@
+/* @flow */
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { View, StyleSheet, TextInput, ScrollView } from "react-native";
+import type { NavigationScreenProp } from "react-navigation";
+
+import SectionEntry from "../../components/SectionEntry";
+import LText from "../../components/LText";
+import { updateAccount } from "../../actions/accounts";
+import type { Account } from "../../types/common";
+
+const mapStateToProps = () => ({});
+const mapDispatchToProps = {
+  updateAccount
+};
+
+class EditConfirmations extends Component<{
+  navigation: NavigationScreenProp<{
+    params: {
+      account: Account
+    },
+    goBack: () => void
+  }>,
+  updateAccount: ($Shape<Account>) => *
+}> {
+  static navigationOptions = {
+    title: "Confirmations"
+  };
+  onConfNumEndEditing = (e: *) => {
+    const { account } = this.props.navigation.state.params;
+    e.nativeEvent.text.length
```

Comment:
as the result is not used in this ternary instruction, I would refer it to be rewritten to

```
if (e.nativeEvent.text.length) {
   updateAccount(...)
}
```